File: POLSKI_SPOJ/C/RNO_DOD.c
#include <stdio.h>

int main() {
    int numberOfCases;
    scanf("%d", &numberOfCases);
    if(numberOfCases > 0 && numberOfCases < 50){
        for(int i = 0; i < numberOfCases; i++){
            int numberOfNumbers;
            scanf("%d\n", &numberOfNumbers);
            int sum = 0;
            for(int j = 0; j < numberOfNumbers; j++){
                int num;
                scanf("%d ", &num);
                sum += num;
            }
            printf("%d ", sum);
        }
    }
    return 0;
}

File: POLSKI_SPOJ/Python/RNO_DOD.py
numberOfCases = input()
for x in range(int(numberOfCases)):
    ammount = input()
    sum = 0
    numbersWithWhitespaces = input()
    numbers = list(numbersWithWhitespaces.split(" "))
    for y in numbers: sum += int(y)
    print(sum)
